### DISCUSSION TITLE
PayloadGenerator.SwissQrCode.Contact deprecated. What do I have to do?

### DISCUSSION THREAD

**Discussion by GrulliGramps** [Shane32/QRCoder#422] [Q&A] [answered]
Hallo everyone, 

after updating QRCoder  to 1.4.3 this code doesn't work anymore: 
_contactCreditor = new SwissQrCode.Contact (name, zipCode, city, country, street, housenr);_
The message is "This constructor is deprecated. Use WithStructuredAddress instead."

I'm new to C# and couldn't figure out how to use this new constructor instead of above statement:
_public static Contact WithStructuredAddress(string name, string zipCode, string city, string country, string street = null, string houseNumber = null);_
Any hints would be appreciated.

Sincerely
Grulli

**Comment by GrulliGramps** [comment, marked as answer]
On StackOverflow I've got an answer. The solution is to us thsi instead of "new ..." :

`contactCreditor = SwissQrCode.Contact.WithStructuredAddress(name, zipCode, city, country, street, housenr);`

**Comment by harshpandey0** [comment]
hey all i am new to this open source can anyone tell me which issues i have to pickup being as a beginner.

i would to love to connect with you all

LinkedIn : https://www.linkedin.com/in/harsh-pandey-2161hp
Skype : https://join.skype.com/invite/w7Fv2iEpdwlS
ShowCase : https://www.showwcase.com/harshpandey9079939
Twitter : https://twitter.com/harshpandey2121
Github : https://github.com/harsh2161

Thankyou
Harsh Pandey
[harshpandey9079@gmail.com](mailto:harshpandey9079@gmail.com)
